app/code/Magento/AdminNotification/composer.json

```diff
@@ -13,6 +13,9 @@
         "magento/module-store": "100.3.*",
         "magento/module-ui": "100.3.*"
     },
+    "require-dev": {
+        "php": "7.0.2|7.0.4|~7.0.6|~7.1.0|~7.2.0"
```

Comment:
Why do we still need 7.0.x?

Reply:
Left it there for now due to having our internal builds running on 7.0. If I remove it, the builds will fail. We should be able to remove it soon.